pandas-stubs/core/series.pyi

```diff
@@ -512,8 +512,8 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
         self,
         axis: AxisType = ...,
         ascending: _bool | Sequence[_bool] = ...,
-        kind: _str | Literal["quicksort", "mergesort", "heapsort"] = ...,
-        na_position: _str | Literal["first", "last"] = ...,
+        kind: Literal["quicksort", "mergesort", "heapsort"] = ...,
```

Comment:
probably should create a type for this since it appears multiple times across files